account_fiscal_position_rule_stock/stock.py

```diff
@@ -75,3 +75,109 @@ def _prepare_invoice(self, cr, uid, picking, partner, inv_type,
         result['fiscal_position'] = picking.fiscal_position and \
             picking.fiscal_position.id
         return result
+
+
+class stock_picking_out(osv.Model):
+    _inherit = "stock.picking.out"
+    _columns = {
+        'fiscal_position': fields.many2one(
+            'account.fiscal.position', 'Fiscal Position',
+            domain="[('fiscal_operation_id','=',fiscal_operation_id)]"),
```

Comment:
Indeed, @rdeheele I couldn't find that field either, could you tell us where does it come from? We kind of have that kind of "operation" thing here in Brazil, so eventually we can build synergies but first we need to understand where it comes from here. Isn't that from some of your custom modules?